tests/test-auth-server/aci/server.go

```diff
@@ -107,6 +109,10 @@ func (h *serverHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	h.sendMsg(fmt.Sprintf("Trying to serve %q", r.URL.String()))
 	switch filepath.Base(r.URL.Path) {
+	case "/":
+		indexHTML := `<meta name="ac-discovery" content="localhost https://localhost/{name}.{ext}">`
```

Comment:
Could you file an issue about dropping explicit handling of "prog.aci" in favor of your fileset stuff?